website/blog/2022-05-10-acf-in-a-project.md

```diff
@@ -0,0 +1,140 @@
+---
+title: ACF in a project
+description: Example of using ACF in your project
+slug: acf-in-a-project
+authors: iobrado
+date: 2022-05-10
+tags: [eightshift, boilerplate, acf, advanced custom fields, theme options]
+hide_table_of_contents: false
+---
+
+If you've been working with WordPress for a while, you must have heard of **_Advanced Custom Fields_** plugin (**_ACF_** for short). While the use of blocks has simplified content editing and meta fields are not as needed anymore, there are still cases where having meta fields is very useful.
+<!--truncate-->
+
+## Ways of registering ACF fields
+
+There are multiple approaches to register ACF fields. The easiest way is through WP Admin interface. But, if you're using Git, this approach isn't the best since the fields are being saved in the database, which means you would have to export and import fields into your staging and production environment each time you make a change.
+
+The recommended way is to register the fields with PHP. There is also an option to register the fields with **_.json_** files, but in Eightshift dev kit, we already have some goodies which will make registering fields with PHP a breeze.
+
+## Getting PHP code export
+
+The easiest way to get the PHP code that you will use for registering ACF fields is by adding those fields through the WP Admin ACF interface first. As an example, I'll add a field group called **_Intro_**, which will contain one text field of the same name. That field group will be displayed only on posts. You can add a few more fields, but for the sake of simplicity, I'll use only one field.
```

Comment:
I feel there is a lot of repetition of the phrase `The X way to Y is ...`
Maybe change up a bit how you start paragraphs.